.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install -r requirements-test.txt
+      - name: Lint with flake8
+        run: |
+          flake8 psutil_home_assistant
+      #- name: Check formatting with black
+      #  run: |
+      #    black psutil_home_assistant --check --diff
```

Comment:
Why is this not enabled?

Reply:
It's disabled because so the content of this PR matches what's already pushed to PyPi 🙈

Reply:
Enabled in #2